### PR TITLE
ADAPT-5848 | @rebeccahongsf | squash email bug

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- squash email bug

# Review By (Date)
- EOD Weds 7/26

# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to https://deploy-preview-691--stanford-alumni.netlify.app
2. Login as `kchrist499` (same pass as `zguan`; see ADAPT-5848)
3. Navigate to https://deploy-preview-691--stanford-alumni.netlify.app/membership/join
4. Select someone else and continue to related contacts page
5. Select either related contact or new contact then continue to form
6. Verify that the email `bharathp@other.stanford.edu` displays (this is the preferred email: "Other Email")
7. Navigate to https://deploy-preview-691--stanford-alumni.netlify.app/travel-study/destinations/finland-2022/finland-reg-form
8. Continue to form
9. Verify that the email `bharathp@other.stanford.edu` displays
10. Review code

# Associated Issues and/or People
- ADAPT-5848

### CODE REVIEW

File: src/components/page-types/membershipFormPage/relatedContactSelection.js [hunk 1]
```diff
@@ -45,11 +45,22 @@ const RelatedContactSelection = (props) => {
   }
 
   // Map related contacts/relationships data to GiveGab ADC values
-  const primaryRegistrantEmail = findEmail(userProfile?.emails);
-  const primaryRegistrantPhoneNumber = findPhoneNumber(
-    userProfile?.phoneNumbers
+  const emailData = fetchEmail(
+    userProfile?.emails,
+    userProfile?.contact?.preferredEmail
   );
```
Comment (rebeccahongsf) [inline src/components/page-types/membershipFormPage/relatedContactSelection.js:51]: The area of shaammme! 👾

File: src/utilities/giveGabVars.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ export const findEmail = (emails, type) => {
     Array.isArray(emails) && type
       ? emails.find((item) => item.type === type)
       : false;
-  return found && { email: found.email, type };
+  return found && { email: found.emailAddress, type };
```
Comment (rebeccahongsf) [inline src/utilities/giveGabVars.js:42]: Another area that was previously overlooked — the emails object has the individual emails keys as `emailAddress` not `email` 😬

**Review by sherakama — APPROVED** [review]
TY

**Review by sherakama — APPROVED** [review]
GTG on green lights

**Pull request merged**